From: Pavel Hrdina <phrdina@redhat.com>
Date: Tue, 14 Jul 2015 15:53:25 +0200
Subject: [PATCH virt-manager] virtinst.cpu: fix copy host cpu definition

Commit cac4ac14 updated cpu features to use XMLBuilder and this change
removes the 'Features' class.  There is no longer any '.names()'
method to return names, so just cycle through all features and use a
name attribute instead.

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1240938

Signed-off-by: Pavel Hrdina <phrdina@redhat.com>
(cherry picked from commit a9b303fb141df5b2d2051c6b4ed489dbb09952f8)
---
 virtinst/cpu.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/virtinst/cpu.py b/virtinst/cpu.py
index b12fd30..fb17bb7 100644
--- a/virtinst/cpu.py
+++ b/virtinst/cpu.py
@@ -104,8 +104,8 @@ class CPU(XMLBuilder):
 
         for feature in self.features:
             self.remove_feature(feature)
-        for name in cpu.features.names():
-            self.add_feature(name)
+        for feature in cpu.features:
+            self.add_feature(feature.name)
 
     def vcpus_from_topology(self):
         """
